client/packages/system/src/Manage/Preferences/EditPage/getPrefKey.tsx

```diff
@@ -0,0 +1,31 @@
+import { PreferencesNode } from '@common/types';
+
+export function getPrefKey(key: string): ClientPrefKey | undefined {
+  // Typescript error if any keys are missing
+  const noMissingKeys: EnsureNoMissingKeys = true;
+
+  if (noMissingKeys) {
+    return SERVER_TO_CLIENT_PREFS[key as MappedServerKey];
```

Comment:
ah this file can be removed now, key mapping handled via the enums 🙏

Reply:
Yeah think this file got added back in the merge conflicts, it can be deleted now!